nginx_proxy/start_nginx.sh

```diff
@@ -42,4 +79,11 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
+# fetch Service Configuration from Service Management if the service name and
+# service version are provided.
+if [[ -n "${SERVICE_NAME}" && -n "${SERVICE_VERSION}" ]]; then
+  /usr/sbin/fetch_service_config.sh \
+    -s "${SERVICE_NAME}" -v "${SERVICE_VERSION}" || exit $?
+fi
```

Comment:
This functionality will move into the nginx.conf generator, so I'd rather not make it over-complicated in start_nginx.sh
Would you be ok with the current all or nothing approach?